app/index.html

```diff
@@ -136,9 +144,10 @@
                     ng-click="trackEvent('download')" ng-href="{{OSX_URL}}" ng-bind-template="{{'MAC' | translate}}">Mac</a>, <a target="_blank" ng-click="trackEvent('download')" ng-bind-template="{{'WINDOWS' | translate}}" ng-href="{{WIN_URL}}">Windows</a>, and <a
                     ng-click="trackEvent('download')" ng-href="{{DEB_URL}}" ng-bind-template="{{'LINUX' | translate}}">Linux</a>.
                 <p class="beta"><span ng-bind-template="{{'PREVIEW_OUR' | translate}}">Preview our </span> <a target="_blank" ng-bind-template="{{'BETA_RELEASE' | translate}}" ng-href="{{BETA_URL}}">beta release</a> 0.2</p></div>
```

Comment:
@atavism I found `0.2` is introduced in [this commit](https://github.com/getlantern/www.getlantern.org/commit/1ed4a80d487b8f4d63006809d3543ae42cb1a42a#diff-b9707fe88797285e93483456d1389b61R138), is it intentionally or not?